docker/server-slim/src/main/docker/image-bootstrap.properties

```diff
@@ -6,4 +6,6 @@ devroot=.
 stdout.toLogBuffer=true
 stderr.toLogBuffer=true
 
-deephaven.console.type=groovy
\ No newline at end of file
+deephaven.console.type=groovy
+deephaven.cache.dir=/cache
+java.io.tmpdir=/cache/tmp
```

Comment:
Can we make this a sub-dir of the host's `/tmp`?

Reply:
Punting - https://github.com/deephaven/deephaven-core/issues/1714